### DISCUSSION TITLE
How to config settings for `alr install` for uploading binaries to embedded boards

### DISCUSSION THREAD

**Discussion by RREE** [alire-project/alire#1757] [Q&A]
I recently came across the command `alr install` and thought that I could use it for uploading binaries. On the AVR platform I use `avrdude`, for RPI Pico / rp2040 I use OpenOCD. Both require lengthy argument lists for uploading `.elf` files to the target. It'd be helpful if I could set the program names and command lines somewhere in the alr settings

**Comment by mosteo** [comment]
`alr install` is using `gprinstall` under the hood. I'm not sure if it can be used for your needs. @Fabien-Chouteau ?

In any case this would be a new feature that we don't currently have.

**Comment by Fabien-Chouteau** [comment]
`gprinstall` is not the tool for that. We could consider an Alire action to override the `install` with a custom command.
